Plugins/GitHub3/GitHub3Plugin.cs

```diff
@@ -267,7 +261,6 @@ public void ConfigureContextMenu(ContextMenuStrip contextMenu)
 
             ToolStripMenuItem toolStripMenuItem = new(string.Format(_viewInWebSite.Text, Name), Icon);
             contextMenu.Items.Add(toolStripMenuItem);
-            toolStripMenuItem.Click += (s, e) => Process.Start(_hostedRemotesForModule.First().Data);
```

Comment:
Why should this be removed?

Reply:
I didn't know if that was an url that we try to open and so after an investigation, I found out that there is no risk to remove for 2 reasons:
* it makes no sense to have this strange action on a menuitem with a submenu. I think it is the remaining of a development test that should have been removed
* all this code is dead code at the moment because that is linked to a feature lost during the transition to the blame inside the browse form. I found the reason but it needs more work and it will be in a following PR...